tests/common.go

```diff
@@ -1882,11 +2045,11 @@ func updateNamespace(in interface{}, namespaceMapping map[string]string) error {
 		namespace := namespaceMapping[specObj.GetNamespace()]
 		specObj.SetNamespace(namespace)
 		return nil
-	} else if specObj, ok := in.(*v1.Service); ok {
+	} else if specObj, ok := in.(*corev1.Service); ok {
 		namespace := namespaceMapping[specObj.GetNamespace()]
 		specObj.SetNamespace(namespace)
 		return nil
-	} else if specObj, ok := in.(*v1.PersistentVolumeClaim); ok {
+	} else if specObj, ok := in.(*corev1.PersistentVolumeClaim); ok {
```

Comment:
Why are we changing to corev1 instead of v1?

Reply:
Older code accidentally imported the same package with both `v1` and `corev1` alias. I removed one of the aliases and kept `corev1` as it is more representative and is less confusing as there are other apis being imported, which also have a `*/*v1` in the package path.